services/nomad/apps/xq-api.nomad

```diff
@@ -0,0 +1,42 @@
+job "xq-api" {
+  datacenters = ["VOID-MIRROR"]
+  namespace = "apps"
+  type = "system"
```

Comment:
need to investigate whether system or service is best for this. nilium says it shouldn't really be an issue to be system.